prow/pilot-presubmit.sh

```diff
@@ -50,6 +50,9 @@ bazel build //...
 echo '=== Go Build ==='
 ./bin/init.sh
 
+echo '=== Code Format Check ==='
+./bin/fmt.sh
```

Comment:
@ldemailly @kyessenov can you please show me for how to use Prow to trigger the presubmit script?